tools/cmake/mbed_set_post_build.cmake

```diff
@@ -5,16 +5,17 @@
 # Sets the post build operation for Mbed targets.
 #
 macro(mbed_set_post_build_operation)
-    add_custom_target(mbed-post-build-bin
-        DEPENDS ${CMAKE_BINARY_DIR}/${APP_TARGET}.bin
+
+    add_custom_target(mbed-post-build-bin-${mbed_target_name}
+        DEPENDS ${CMAKE_CURRENT_BINARY_DIR}/${APP_TARGET}.bin
```

Comment:
This will be fixed also separately, it got in 3 days ago, right after APP_TARGET was cleaned up. It affects currently only post build targets. We will look at this soon.